hudi-common/src/main/java/org/apache/hudi/keygen/constant/KeyGeneratorOptions.java

```diff
@@ -54,6 +54,13 @@ public class KeyGeneratorOptions extends HoodieConfig {
           + "Actual value will be obtained by invoking .toString() on the field value. Nested fields can be specified using\n"
           + "the dot notation eg: `a.b.c`");
 
+  public static final ConfigProperty<String> SECONDARYKEY_FIELD_NAME = ConfigProperty
+      .key("hoodie.datasource.write.secondarykey.field")
```

Comment:
column or field, because in `HoodieMetadataConfig`, we have `*.index.secondary.column`.

Reply:
made it `.column` to keep consistent with HoodieMetadataConfig.